File: RepoTransBench/source_projects/Java/googlearchive_android-BluetoothLeGatt/Application/src/test/java/com/example/android/bluetoothlegatt/BluetoothLeServiceTestBasic.java
package com.example.android.bluetoothlegatt;

import org.junit.Test;
import static org.junit.Assert.*;

public class BluetoothLeServiceTestBasic {
    @Test
    public void testStaticFieldsNotNull() {
        assertNotNull(BluetoothLeService.ACTION_GATT_CONNECTED);
        assertNotNull(BluetoothLeService.ACTION_GATT_DISCONNECTED);
        assertNotNull(BluetoothLeService.ACTION_GATT_SERVICES_DISCOVERED);
        assertNotNull(BluetoothLeService.ACTION_DATA_AVAILABLE);
        assertNotNull(BluetoothLeService.EXTRA_DATA);
        assertNotNull(BluetoothLeService.UUID_HEART_RATE_MEASUREMENT);
    }
}

File: RepoTransBench/target_projects/Java/C#/googlearchive_android-BluetoothLeGatt/tests/original/BluetoothLeServiceTestBasic.cs
using Xunit;
using BluetoothLeGatt;

namespace Tests.Original
{
    public class BluetoothLeServiceTestBasic
    {
        [Fact]
        public void TestStaticFieldsNotNull()
        {
            Assert.NotNull(BluetoothLeService.ACTION_GATT_CONNECTED);
            Assert.NotNull(BluetoothLeService.ACTION_GATT_DISCONNECTED);
            Assert.NotNull(BluetoothLeService.ACTION_GATT_SERVICES_DISCOVERED);
            Assert.NotNull(BluetoothLeService.ACTION_DATA_AVAILABLE);
            Assert.NotNull(BluetoothLeService.EXTRA_DATA);
            Assert.NotNull(BluetoothLeService.UUID_HEART_RATE_MEASUREMENT);
        }
    }
}